### ISSUE TITLE
Unable to install groups

### ISSUE THREAD

**Issue by justin8** [oshazard/apacman#51]
When installing a package group, it tries to default to the first package, as though multiple packages provide a single dependency, instead of installing a group.

Current output:
```
:: There are 13 packages that provide ladspa-plugins:
1) cmt
2) swh-plugins
3) amb-plugins
4) blop
5) caps
6) fil-plugins
7) g2reverb
8) mcp-plugins
9) pvoc
10) rev-plugins
11) tap-plugins
12) vco-plugins
13) wah-plugins

Enter a selection (default=1):
```

Expected output:
```
:: There are 13 members in group ladspa-plugins:
:: Repository extra
   1) cmt  2) swh-plugins
:: Repository community
   3) amb-plugins  4) blop  5) caps  6) fil-plugins  7) g2reverb  8) mcp-plugins
   9) pvoc  10) rev-plugins  11) tap-plugins  12) vco-plugins  13) wah-plugins

Enter a selection (default=all):
```

**Comment by oshazard** [comment]
Swapping the order of the `providedinpacman` and `existsinpacmangroup` calls did the trick, heh

**Comment by oshazard** [comment]
Fixed in  2abca2e

**Example: installing a group**
```
apacman -S ladspa-plugins
Pacman Targets    (1): ladspa-plugins

:: There are 13 members in group ladspa-plugins:
:: Repository extra
   1) cmt  2) swh-plugins
:: Repository community
   3) amb-plugins  4) blop  5) caps  6) fil-plugins  7) g2reverb
   8) mcp-plugins  9) pvoc  10) rev-plugins  11) tap-plugins  12) vco-plugins
   13) wah-plugins

Enter a selection (default=all): 
```

**Example: installing a virtual package**
```
apacman -S ttf-font
:: There are 10 packages that provide ttf-font:
1) noto-fonts
2) ttf-bitstream-vera
3) ttf-croscore
4) ttf-dejavu
5) ttf-freefont
6) ttf-linux-libertine
7) ttf-oxygen
8) ttf-droid
9) ttf-liberation
10) ttf-ubuntu-font-family

Enter a selection (default=1):
```
